### PR TITLE
fix #1285, fix travis deploy build script

### LINKED ISSUE #1285: fix travis deploy script

### PR DESCRIPTION
**Pull request by clockfly**
[Review on Reviewable](https://reviewable.io/reviews/intel-hadoop/gearpump/1286)

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -186,7 +186,10 @@ object Build extends sbt.Build {
           val packagePath = s"output/target/gearpump-pack-${version.value}.tar.gz"
           val target = s"target/binary.gearpump.tar.gz"
           println(s"[Travis-Deploy] Move file $packagePath to $target")
-          new File(packagePath).renameTo(new File(target))
+          val file = new File(packagePath)
+          val targetFile = new File(target)
+          targetFile.getParentFile.mkdirs()
+          file.renameTo(targetFile)
```
Comment (manuzhang) [inline project/Build.scala]: why renamed to this target ? Will the target file contain any version info ?

Comment (clockfly) [inline project/Build.scala]: Travis deploy only allow static package name, which means version should not exist in the package name.

Comment (manuzhang) [inline project/Build.scala]: can we use the `file_glob` option documented [here](https://github.com/travis-ci/dpl#github-releases)

Comment (clockfly) [inline project/Build.scala]: Great, looks like they added this new feature.

**Pull request closed without merging**